actions/ActionTypes.js

```diff
@@ -0,0 +1 @@
+export const ADD_THOUGHT = 'add_thought';
```

Comment:
While unconventional, I prefer using string literals in camelCase for action types. I have not noticed any benefit to action type definitions as mistyped action types can throw in the reducer.